modules/sts-mode-installing-manual-run-installer.adoc

```diff
@@ -5,7 +5,39 @@
 [id="sts-mode-installing-manual-run-installer_{context}"]
 = Running the installer
 
-* Run the {product-title} installer:
+.Prerequisites
+
+* Create the `install-config.yaml` file for your installation.
+
+.Procedure
+
+. From the installation directory that contains your `install-config.yaml` file, configure the  Cloud Credential Operator (CCO) to run in manual mode:
++
+[source,terminal]
+----
+$ echo “credentialsMode: Manual” >> install-config.yaml
+----
+
+. Create the required {product-title} installation manifests:
++
+[source,terminal]
+----
+$ openshift-install create manifests
+----
+
+. Copy the manifests that `ccoctl` generated to the manifests directory that the installation program created:
++
+----
+$ cp /<path_to_ccoctl_output_dir>/manifests/* ./manifests/
+----
+
+. Copy the private key that the `ccoctl` generated in the `tls` directory to the installation directory:
++
+----
+$ cp -a /<path_to_ccoctl_output_dir>/tls .
```

Comment:
In another part of the doc, you italicized the path variable so maybe it should be italicized here and in step 3?

Reply:
yes - and it didn't even have a source tag! oops